metrics/ci_integrations/test/source/github_actions/adapter/github_actions_source_client_adapter_test.dart

```diff
@@ -651,6 +806,30 @@ void main() {
       },
     );
 
+    test(
+      ".fetchBuildsAfter() returns an empty list if the client returns workflow run jobs with the queued status",
+      () async {
+        final queuedJob = testData.generateWorkflowRunJob(
+          status: GithubActionStatus.queued,
+        );
+
+        final workflowRunJobsPage = WorkflowRunJobsPage(values: [queuedJob]);
+
+        whenFetchWorkflowRuns(
+          withJobsPage: workflowRunJobsPage,
+        ).thenSuccessWith(defaultRunsPage);
+
+        final firstBuild = testData.generateBuildData(buildNumber: 1);
```

Comment:
```suggestion
        final buildData = testData.generateBuildData(buildNumber: 1);

        whenFetchWorkflowRuns(
          withJobsPage: workflowRunJobsPage,
        ).thenSuccessWith(defaultRunsPage);
```
What do you think? Please, apply this suggestion to all similar situations.